cmd/install/install.go

```diff
@@ -20,20 +27,23 @@ var (
 	ErrCurlMissing     = errors.New("please install curl and set it in path")
 	kubectlPath        string
 	argocdPath         string
-	defaultKubectlPath = "/usr/local/bin/kubectl"
-	defaultArgocdPath  = "/usr/local/bin/argocd"
+	kubeconfigPath     string
 	Yellow             = color.New(color.FgHiYellow).SprintFunc()
 	Green              = color.New(color.FgGreen).SprintFunc()
 	Red                = color.New(color.FgRed).SprintFunc()
+	white              = color.New(color.FgHiWhite).SprintFunc()
+	capiCoreProvider   string
+	infraProviders     []string
```

Comment:
So no providers are specified by default? If the user realizes this after the fact, can he run `install` again with the right options?

Reply:
We're not installing different provider components because all of them require a form of credential, and the steps of obtaining that credential is different for every provider. Please check this https://github.com/arlonproj/arlon/pull/214#discussion_r971076033 conversation above.

Reply:
Update: the new changes can make it possible to install providers afterwards
https://github.com/arlonproj/arlon/pull/214#issuecomment-1249031554